allium/lib/prometheus_metrics.py

```
@@ -169,6 +169,9 @@ def _build_aroi_relay_rows(relays: List[dict], fp_to_family: Callable[[str], str
     rows = []
     state_counts = {state: 0 for state in _AROI_STATE_VALUES}
     configured_count = 0
+    # B7.1: per-version state count (state, ciissversion) -> count.
+    # Cardinality bounded: |state| × |ciissversion| ≤ 4 × 5 = 20 series.
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)